Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -348,7 +353,8 @@ class BrowserViewController: UIViewController {
                     if let readerMode = self.tabManager.selectedTab?.getHelper(name: ReaderMode.name()) as? ReaderMode where readerMode.state == .Active {
```

Comment:
Nope, no problems. Looks like that line was removed accidentally, so you can just ignore my comment.